blog/2023-09-06-consensus.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Consensus Team Update
+slug: 2023-09-06-consensus
+authors: dnadales
+tags: [consensus]
+hide_table_of_contents: false
+---
+
+## High level summary
+
+We were able to successfully run the system-level benchmarks for the UTxO-HD implementation, for the first time. There was an important regression in block forging performance that will have to be addressed before UTxO-HD is released. We also revisited the implementation of our query processing logic, which was needed to address the performance regression found in the query-by-address command. The preliminary performance results show that now the performance of this query is on-par with the Cardano baseline version, but we need further confirmation.
+On the Genesis front, we presented the grinding-aware safety argument for the proposed historical Cardano Genesis windows to the IOG Researchers.
+Version `8.3.0-pre` of `cardano-node` is releasing 2023 September 5.
```

Comment:
This line seems a little out of place without the detail that the rotating release engineer is currently from the Consensus team.

Reply:
What about:

> The Consensus release engineer finished his rotation: version `8.3.0-pre` of `cardano-node` is releasing 2023 September 5.